### DISCUSSION TITLE
案例：数据新闻的另一种视角与实现——《转角遇到爱》作品分析

### DISCUSSION THREAD

**Discussion by Tiffany0718** [data-journalism/data-journalism.github.io#43] [Show and tell]
作品链接：https://h5.thepaper.cn/html/zt/2018/08/seekinglove/index.html

简介：获得2018年SND（美国新闻媒体视觉设计协会）最佳数字设计铜奖。选一个晴天的周日，从上海人民广场地铁站9号口出门，左手边就是闻名全国的人民广场相亲角。五六十岁模样的大叔大妈们带着伞和小板凳，在这里为他们的晚辈寻觅一份姻缘。澎湃新闻（www.thepaper.cn）和姐妹英文媒体“第六声”的数据记者花费了六个周末的时间，收集了874份相亲广告。从中可以读出关于618位女士和256位男士的觅爱故事。

### **创作缘起**
1、“相亲”“催婚”的话题是当下热门的社会话题，媒体也会聚焦于此，且主创团队的成员也有被催婚的经历；
2、目前这个题材被做的很多，但是在数据方面挖掘的并不多。
3、方向确定：数据新闻编辑部试图把相亲角每一把伞上的信息收集起来，在对数据中的个人信息进行脱敏后，从微观的视角来看看这些信息背后有什么特点。编辑部把这些文本数据结构化后，想看看在相亲角这个地方，人们（更多是家长们）是如何定义自己（的孩子），如何描述想要的对象，以及人和人之间价值是在一个什么样的基础上进行交换的。

### **内容介绍**
**引入：上海相亲角的俯瞰图**
-记者前往上海人民广场相亲角6次收集了874份征婚启事
-地图显示上海相亲角的位置 
![image](https://user-images.githubusercontent.com/61171039/136802723-7d0993dd-4946-46b8-bba5-38f7f654b08b.png)

**数据呈现**
收集到的874份征婚广告经过脱敏处理，每则广告由一把伞来表示，广告的内容按照_【性别、年龄、学历、户口、退休、海外】_几类条件呈现为不同颜色的伞沿。
![image](https://user-images.githubusercontent.com/61171039/136802899-2413bdd6-e006-46e6-8a91-1f38ba0da301.png)

874份征婚广告代表着874个人，按照性别，男士用蓝色的小球来代替，女士用红色的小球代替，用柱状图、桑基图、条形图等对比分析了相亲者的_【年龄、身高、婚姻状况、出生地、户口、学历、工作、年薪、外貌、性格、兴趣爱好、家庭状况、房产】_情况。
![image](https://user-images.githubusercontent.com/61171039/136803532-0493e2d3-6f4b-40da-9797-c3763d077a1f.png)
![image](https://user-images.githubusercontent.com/61171039/136803560-ea0f178d-0937-438f-afc4-73e9cd44bad1.png)
![image](https://user-images.githubusercontent.com/61171039/136803893-dfa15f29-0278-4d1f-8807-cd55614a6a77.png)
![image](https://user-images.githubusercontent.com/61171039/136803903-a7cbe83c-8593-44aa-85f1-4bf4de0aa83d.png)
![image](https://user-images.githubusercontent.com/61171039/136803943-980d2a45-5f5d-4c00-a1ba-31d15dc77a27.png)

**数据分析**
-以性别为基础分类
-例举典型、展示信息填写率
-要求“门当户对”的比例：没有直接给出数据，但用深色小球表示
![image](https://user-images.githubusercontent.com/61171039/136803975-ef7931f6-c6e7-4dc8-a02e-b7002a74a864.png)

-相亲人年龄：女性更加年轻，男性年龄分布更加平均
-相亲人对对方年龄的要求：男性更偏好比自己年轻的，女方对年龄的要求比较分散
-对房产的要求：男性大多不要求对方有房产，有房产要求的男性基本自己都有房产，展示了房产条件与要求的细节。
-户口要求：最能看出资源互换的意味。
-兴趣爱好：填写率过低，内容大多集中于“无不良嗜好”。
-性格的自我描述：存在严重的同质化。

**升华主题**
【展现事实，提出问题】
作品最后引用孙沛东教授对于“相亲”这一现象的描述：“以关爱为情感基础的相亲活动，反而成为一种变相的商品交易活动...真正具有情感内容的过程——谈情说爱的过程——被舍弃了”这一点可以在极低的兴趣爱好填写率和高度同质化的性格描述中得到印证。相亲角的男男女女，多数更加关注物质条件，很少关注未来的另一半、甚至自己的精神世界。“

上海政府网曾有报道称，这里的成功配对不足1%。从澎湃新闻的相亲角数据库来看，相亲角的男女至少有十对是可以互相精确匹配的。TA们，有遇见彼此吗？”这个作品想展现的不仅是相亲角中男男女女的自身情况和相亲要求，更多地是想唤起对于当代社会人际关系的思考。

### **可视化效果分析**
**调查数据形象呈现：易读性**
报道巧妙地用伞来象征相亲角的征婚广告，每把伞代表一则广告，并用6种不同的伞沿颜色表示不同的条件类型。最终将他们的调查结果用出现的伞沿颜色显示出来——伞沿有颜色，则表示该广告中出现了相应条件类型的描述。
![image](https://user-images.githubusercontent.com/61171039/136804196-913e528e-8818-4e12-990b-d013717e533c.png)

报道中除了列举了每则广告的类型条件，还以图文结合的形式展现了整体的征婚条件。
![image](https://user-images.githubusercontent.com/61171039/136804213-e8a7378c-132d-472f-bb66-6bbf73667210.png)
![image](https://user-images.githubusercontent.com/61171039/136806577-5f1a1c6d-8858-41d4-8fb4-1ec9c4e73dc8.png)

**对比分析**
涉及需要对比分析的数据，大多数报道会采用条形图或百分比数据等方式来展示。该报道将每一个调查数据视为一个小圆点，并用不同颜色划分性别，用不同颜色深浅划分满足的条件类型。
![image](https://user-images.githubusercontent.com/61171039/136804260-a7680407-9934-4d11-a546-7a50939368fe.png)
![image](https://user-images.githubusercontent.com/61171039/136804285-859d2d0e-ee17-4578-8a3b-b89440f1c460.png)

报道以运动的小球为主，以数轴、折线、文字、词条多形式为辅相结合的方式来展示并进行对比分析，既表现了每一个类别的具体数据，又方便读者对各种类别的数据进行直观的对比。
![image](https://user-images.githubusercontent.com/61171039/136804321-ac638dcd-b967-4993-b2ae-526af8a3ed8f.png)

**数据动图：动态呈现自身条件与需求差异**
报道使用数据动图，将抽象的数据含义用丰富的可视化图表表现得更加具象，同时也通过动态的形式来反映上海相亲角的征婚人群自身条件的差异以及需求的差异。
![image](https://user-images.githubusercontent.com/61171039/136804371-900e3bbf-7be7-4f98-92ae-b3dc8f96c6dc.png)
![image](https://user-images.githubusercontent.com/61171039/136804400-dba23bc5-b423-44f0-b3cf-faea73e829bd.png)

**交互体验——参与和互动**
报道除了将数据直观地展示给读者，也会让读者自主选择。数据的表现形式也不仅仅是数字或图形，同时也注重用户对数据的体验。报道中的数据除了有具体量化的表现形式，还利用了交互式的伞状图，通过点击不同的伞就可以看到不同的征婚广告，更加具有沉浸感、交互性和想象力，让读者能够产生更多互动，增强参与感。
![image](https://user-images.githubusercontent.com/61171039/136804482-2984caa5-e1ce-43c3-9901-eea8aadc391e.png)

### **技术实现**
**数据脱敏**
《转角遇到爱》中的一大特点就是对文本数据的处理，其中进行了隐私内容脱敏处理。这个过程的主要目标就是，这几种信息合在一起，不让涉及到的相亲男女周围的人察觉到他们是谁。比如，有个人是“市级医院XX主任” ，当时大家觉得已经去敏了，但吕老师就觉得，这条信息和其他信息放在一起，他周围人的可能一下就会认出来所以就又做了一步“医院科主任”这样范围就更模糊了。

这就是我们常说的数据脱敏。数据脱敏是指对某些敏感信息通过脱敏规则进行数据的变形，实现敏感隐私数据的可靠保护。在涉及客户安全数据或者一些商业性敏感数据的情况下，在不违反系统规则条件下，对真实数据进行改造并提供测试使用，如身份证号、手机号、卡号、客户号等个人信息都需要进行数据脱敏。这样就可以在开发、测试和其它非生产环境以及外包环境中安全地使用脱敏后的真实数据集。

如果大家对数据脱敏的做法感兴趣，或者想了解如何进行数据脱敏处理不妨参考下面的链接。
数据脱敏的6种方案_胖大海-CSDN博客_数据脱敏的常用方法：https://blog.csdn.net/u010517268/article/details/112864274

**Swiper**
我们经常在看很多H5的数据新闻时，会看到滑动滚播的效果，这种效果很显然也在《转角遇到爱》中获得了实现。这其实和一款基于JAVA SCRIPT的插件有关。这个小魔法就是Swiper。

swiper是一款轻量级的轮播图插件，不仅支持pc端更是为移动端而生，用它可以快速地做出一个轮播图，或者扩展使其做出复杂的轮播效果。Swiper常用于移动端网站的内容触摸滑动。Swiper能实现触屏焦点图、触屏Tab切换、触屏轮播图切换等常用效果。Swiper开源、免费、稳定、使用简单、功能强大，是架构移动终端网站的重要选择！如果想了解如何使用Swiper实现自己想要的效果，可以参考以下的链接。

Swiper中文网-轮播图幻灯片js插件,H5页面前端开发：https://swiper.com.cn/

### **总结**
这个作品向我们诠释了数据新闻的另一种可能。这也就迎合了该作品的主创人的初期想法。“数据新闻，不一定要是非常数字化的东西，其实在我们的生活当中有很多可能被我们忽略到，但是放在一起就是一个很有趣的数据点，大家可能会觉得原来这个题可以这样子做，用数据的方法去做。”邹熳云分享道。这其实给我们做数据新闻的时候也提供了另一种思考的路径，什么样的东西可以成为选择数据新闻的选题，又可以怎样做？这些都是再去省思的点。

但是该作品的数据收集与数据分析部分并不是十全十美的。首先，数据收集的这874份相亲广告存在很强的随机性，所以并不能深入挖掘现象与问题，最多只能算是一种尝试。主创团队承认八百多份的样本量，还不足以证明任何假说。但是，这足够让我们对现在的相亲市场乃至婚恋市场进行一场深入的观察。其次，针对问题我们才找到文本数据来支撑分析，然而这个前提在于文本内容本身符合我们方向与要求。但是，这些相亲广告可能由父母或亲友代笔，也可能由子女自己写作，所以我们无法确定双方的看法各占多少。

**Comment by chengjun** [comment]
![Capture](https://user-images.githubusercontent.com/543384/136884567-5057fd62-f575-464b-8869-a22dd566f2ea.PNG)

[db_new.csv](https://github.com/data-journalism/data-journalism.github.io/files/7326602/db_new.csv)
